### PR TITLE
Add Custom Cohorts Documentation 

### PR DESCRIPTION
**Pull request by vivbak**
- What is a cohort 
- What is a cohort template 
- How do I create them

### CODE REVIEW

**Comment by amyminiter** [comment]
This all makes sense. You've done a great job of summarising what you communicated in the CPG-wide presentation and in the data managers meeting ✅

**Review by amyminiter — APPROVED** [review]

File: metamist/cohorts.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+
+## How can I create a cohort?
+
+### Recommended
+
+* Use the create custom cohort script [create_custom_cohort.py](https://github.com/populationgenomics/metamist/blob/dev/scripts/create_custom_cohort.py)
+* Input all specified parameters, including your desired cohort name, template ID, or criteria.
+
+Example From Existing Template:
```
Comment (illusional) [inline metamist/cohorts.md:43]: Why not just run the script locally?

**Review by illusional — APPROVED** [review]

File: metamist/cohorts.md (hunk 1 above)
Comment (vivbak) [reply metamist/cohorts.md:43]: As in, why use the analysis-runner?

Comment (illusional) [reply metamist/cohorts.md:43]: Yeah, the endpoint should only require read-access (until better metamist permissions are available) right?

**Comment by vivbak** [comment]
Just awaiting explicit approval from @michael-harper and @EddieLF :) Let me know if you have any qs!

**Review by michael-harper — APPROVED** [review]
Very clear - looking forward to using this in PA! Awesome! 👏

**Review by EddieLF — APPROVED** [review]
Looks good to me :shipit:

**Pull request merged**